crates/outline_panel/src/outline_panel.rs

```diff
@@ -4832,7 +4832,7 @@ fn file_name(path: &Path) -> String {
 
 impl Panel for OutlinePanel {
     fn persistent_name() -> &'static str {
-        "Outline Panel"
+        "OutlinePanel"
```

Comment:
Note: those 2 `persistent_name` changes spoil the DB restoration, at least the first time the new version is out.

Seems redundant to do?